README.md

```diff
@@ -42,6 +42,7 @@ Or, in the simple case of just a `from` with the default destination, you can us
 | `ignore` | N | [] | Additional globs to ignore for this pattern |
 | `transform` | N | function(content, path) {<br>&nbsp;&nbsp;return content;<br>} | Function that modifies file contents before writing to webpack |
 | `force` | N | false | Overwrites files already in compilation.assets (usually added by other plugins) |
+| `writeWhenEmit` | N | false | If set to `true` will emit to build folder and memory in combination with `webpack-dev-server` |
```

Comment:
Can we find a terser name for this `write || emit`?

Reply:
@michael-ciniawsky yep, write seems good for me